paddle/fluid/eager/grad_node_info.cc

```diff
@@ -263,6 +263,69 @@ void GradNodeBase::SetGradOutMeta(const paddle::Tensor& fwd_in,
   }
 }
 
+/*
+special func for matmul_double_grad etc. dx exists when x and y exists,
+if stop_gradient of y is true, dy is None who is matmul_grad's out_put, ddy is
+None, so dx = ddy * dout should be None.
+*/
+void GradNodeBase::SetGradOutMeta(const paddle::Tensor& fwd_in,
+                                  const AutogradMeta* fwd_in_other_meta,
```

Comment:
forward other in

Reply:
modify it as "forward_out_meta" because it is forward api's output